blocks/modal/modal.js

```diff
@@ -45,6 +47,11 @@ export async function createModal(contentNodes, config) {
     document.body.classList.remove('modal-open');
     block.remove();
     document.dispatchEvent(new CustomEvent('modal-closed', { bubbles: true, detail: config }));
+    sendDigitalDataEvent({
```

Comment:
since we are already dispatching event on model closed & open. we can listener in analytics code and send events